.travis.yml

```diff
@@ -2,17 +2,17 @@ language: python
 
 python:
   - 3.5
+  - 3.8
 
 env:
-- TOXENV=django111
-- TOXENV=django20
-- TOXENV=django21
 - TOXENV=django22
 
 matrix:
   include:
   - python: 3.5
     env: TOXENV=quality
+  - python: 3.8
```

Comment:
We can move quality to env section too and remove matrix altogether.

Reply:
Moved the `quality` to `env` section and removed the `matrix`.